backend/.env.staging.example

```
@@ -0,0 +1,28 @@
+# Создать пользователя: psql -U postgres -d tasktime -f scripts/mcp-db-user.sql
+#
+# Режим запуска:
+#   HTTP transport (Docker на сервере):
+#     MCP_SERVICE_TOKEN=<token> MCP_AGENT_PASSWORD=<pwd> docker compose --profile mcp up -d mcp-flow-universe
+#
+#   Stdio (локальная разработка, SSH-туннель на БД):
+#     ssh -L 5434:localhost:5432 root@5.129.242.171 -N &
+#     ssh -L 3001:localhost:3000 root@5.129.242.171 -N &
+#     MCP_ENV=staging npm run mcp
+DATABASE_URL="postgresql://flowuniverse_mcp:CHANGE_ME@localhost:5434/tasktime?schema=public"
+REDIS_URL="redis://localhost:6380"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)